components/suggestion/switch_channel_provider.tsx

```diff
@@ -499,7 +552,7 @@ export default class SwitchChannelProvider extends Provider {
         });
     }
 
-    userWrappedChannel(user, channel) {
+    userWrappedChannel(user: UserProfile, channel: Channel | null | undefined): WrappedChannel {
```

Comment:
Can specify undefined only here but AFAIK channel should always be present here?